lib/canvas-motion.js

```diff
@@ -28,8 +30,14 @@ CanvasMotion.prototype = {
   }
 };
 
-function moveFishX(fish, round) {
-  fish.x += (fish.velocity * round);
+function moveFishX(fish, round, p5Amplitude) {
+  // console.log("moveFishX " + p5Amplitude);
```

Comment:
Keep this here for time being. It's a good indication that it's getting passed all the way there.